src/main/java/hudson/plugins/buildblocker/BlockingJobsMonitor.java

```diff
@@ -147,18 +154,20 @@ private SubTask checkComputerForRunningBuilds(Computer computer) {
         return null;
     }
 
-    public SubTask checkNodeForRunningBuilds(Node node) {
+    @WithBridgeMethods(value=SubTask.class)
+    public Job checkNodeForRunningBuilds(Node node) {
         if (node == null) {
             return null;
         }
         return checkComputerForRunningBuilds(node.toComputer());
     }
 
-    private SubTask checkForPlannedBuilds(Queue.Item item, List<? extends Queue.Item> buildableItems) {
+    @WithBridgeMethods(value=SubTask.class)
+    private Job checkForPlannedBuilds(Queue.Item item, List<? extends Queue.Item> buildableItems) {
         for (Queue.Item buildableItem : buildableItems) {
             if (item != buildableItem) {
                 for (String blockingJob : this.blockingJobs) {
-                    AbstractProject project = (AbstractProject) buildableItem.task;
+                    Job project = (Job) buildableItem.task;
```

Comment:
:bug: this is an unsafe cast. Even in Workflow itself there is an example of a `Task` which is not a `Job` (`ExecutorStepExecution.PlaceholderTask`)!